CMakeLists.txt

```diff
@@ -220,6 +220,8 @@ us_cache_var(CMAKE_DEBUG_POSTFIX d STRING "Executable and library debug name pos
 
 us_cache_var(US_ENABLE_THREADING_SUPPORT ON BOOL "Enable threading support")
 us_cache_var(US_ENABLE_TSAN OFF BOOL "Enable tsan (thread sanitizer)" ADVANCED)
+us_cache_var(US_ENABLE_ASAN OFF BOOL "Enable asan (address sanitizer)" ADVANCED)
+us_cache_var(US_ASAN_USER_DLL "" STRING "Path to asan dll" ADVANCED)
```

Comment:
if this variable is only for ASan support on Windows, I suggest making that explicit in the variable description. So something like "Path to Windows Asan DLL"